app/_locales/en/messages.json

```diff
@@ -4155,6 +4155,18 @@
   "onboardingImportWallet": {
     "message": "I have an existing wallet"
   },
+  "onboardingLoginFooter": {
+    "message": "By continuing, I agree to MetaMask’s"
+  },
+  "onboardingLoginFooterAnd": {
```

Comment:
Use locale templating here instead of separating the whole sentence. Like this `By continuing, you agree to MetaMask’s $1 and $2.`